arch/arm/boot/dts/overlays/pitft28-resistive-overlay.dts

```diff
@@ -64,6 +64,9 @@
 			};
 
 			pitft_ts@1 {
+                /* needed to avoid dtc warning */
```

Comment:
Please use tabs to indent. (That should be the case in the other 2 files, but at least they are self-consistent in using spaces throughout).